tensorflow/lite/micro/tools/requantize_flatbuffer.py

```diff
@@ -0,0 +1,196 @@
+# Copyright 2023 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+"""An experimental tool to requantize a int8 activation, int8 weight LSTM based model to int16 activation, int8 weight
+
+Steps: 
+1. Convert the trained model to int8 using the TFLite converter. See https://www.tensorflow.org/lite/performance/post_training_quantization#full_integer_quantization
+2. Use this tool to requantize the int8 model to int16.
+3. Check if the requantized model match the expectation (e.g., read the conversion printout, perform inference tests)
+
+The conversion process: 
+1. Requantize the ops specified in _COMPLEX_OP_REQUANTIZE_REGISTRATION using the registered function. Bias type conversion (int32 to int64) only happens here. 
+2. Requantize all non-constant tensors with int8 type to int16 (and fix the quantization parameters)
+
+Run:
+bazel build tensorflow/lite/micro/tools:requantize_flatbuffer
+bazel-bin/tensorflow/lite/micro/tools/requantize_flatbuffer
+--int8_model_path=".tflite file path"` --save_path="save path"
+
+CAVEAT: 
+1. Use this tool ONLY for models that contain the LSTM layer. All other models should use the standard tflite conversion process.
+2. This is an experimental tool. ALWAYS check if the converted model matches your expectation
+3. Add the custom op requantization function for complex ops (e.g., convolution). 
+4. We assume ops not in _COMPLEX_OP_REQUANTIZE_REGISTRATION only have activation tensors (i.e. no weights and bias). Check the quantized model performance if you add additional ops to _TESTED_SIMPLE_OPS 
+
+"""
+import os
+
+import numpy as np
+from absl import app
+from absl import flags
+from absl import logging
+
+from tflite_micro.tensorflow.lite.tools import flatbuffer_utils
+import tflite_micro.tensorflow.lite.micro.tools.requantize_flatbuffer_utils as utils
```

Comment:
nit: don't shorten to utils (that's too generic).

https://google.github.io/styleguide/pyguide.html#224-decision

```py
from tflite_micro.tensorflow.lite.micro.tools import requantize_flatbuffer_utils
```

Reply:
Done.